downloads.md

```diff
@@ -53,17 +53,6 @@ Spark artifacts are [hosted in Maven Central](https://search.maven.org/#search%7
 ### Installing with PyPi
 <a href="https://pypi.python.org/pypi/pyspark">PySpark</a> is now available in pypi. To install just run `pip install pyspark`.
 
-### Spark Source Code Management
```

Comment:
Could you move this or some version of it to `contributing.md` instead? That page doesn't seem to actually cover this.

Reply:
Thanks @vanzin for the feedback, I have moved the cloning instructions to `contributing.md` and the info about nightly builds is already in `developer-tools.md`